python/lsst/summit/utils/astrometry/fastStarTrackerAnalysis.py

```diff
@@ -50,6 +54,29 @@
     "plotSourcesOnImage",
 )
 
+KNOWN_CAMERAS = ("regular", "wide", "fast")
+
+
+@dataclass(frozen=True)
+class StarTrackerCamera:
+    """A frozen dataclass for StarTracker camera configs"""
+
+    cameraType: str
+    suffix: str
+    suffixWithSpace: str
+    doAstrometry: bool
+    cameraNumber: int
+    snr: float
+    minPix: int
+    brightSourceFraction: float
+    scaleError: float
+    doSmoothPlot: bool
+
+
+narrowCam = StarTrackerCamera("regular", "", "", True, 102, 5, 25, 0.95, 5, True)
```

Comment:
Again I find it slightly confusing to have regular/narrow but I assume there is a reason for it.

Reply:
No no, you're right, this was also supposed to be renamed everywhere.